libs/api/domains/auth-admin/src/lib/tenant/tenant-merged-environment.resolver.ts

```diff
@@ -0,0 +1,15 @@
+import { UseGuards } from '@nestjs/common'
+import { Parent, ResolveField, Resolver } from '@nestjs/graphql'
+
+import { IdsUserGuard } from '@island.is/auth-nest-tools'
+
+import { TenantMergedEnvironment } from './models/tenant-merged-environment.model'
+
+@UseGuards(IdsUserGuard)
+@Resolver(() => TenantMergedEnvironment)
+export class TenantMergedEnvironmentResolver {
+  @ResolveField('id', () => String)
+  resolveId(@Parent() tenant: TenantMergedEnvironment) {
```

Comment:
Fyi I’m pretty sure this is not needed because the id is unique within the environment typename.